source/textInfos/__init__.py

```diff
@@ -472,17 +477,26 @@ def getTextInChunks(self, unit):
 			yield chunkInfo.text
 			unitInfo.collapse(end=True)
 
-	def getControlFieldSpeech(self, attrs, ancestorAttrs, fieldType, formatConfig=None, extraDetail=False, reason=None):
+	def getControlFieldSpeech(
+			self, attrs, ancestorAttrs, fieldType, formatConfig=None, extraDetail=False, reason=None
+	) -> SpeechSequence:
 		# Import late to avoid circular import.
 		import speech
-		return speech.getControlFieldSpeech(attrs, ancestorAttrs, fieldType, formatConfig, extraDetail, reason)
+		sequence = speech.getControlFieldSpeech(
+			attrs, ancestorAttrs, fieldType, formatConfig, extraDetail, reason
+		)
+		# remove empty strings or empty sequences which originate from speech.getControlFieldSpeech

```

Comment:
I guess we need to do this here because `# remove empty strings or empty sequences which originate from speech.getControlFieldSpeech` has some early returns?

Reply:
I'm not entirely sure. I might have to go back and look at this again. It may have been a result of only being partially finished converting functions.